### ISSUE TITLE
Final Report Peer Review

### ISSUE THREAD

**Issue by szhengyi16** [tc457/COVID_US_Economy#9]
The team proposed to study which COVID-related factors could drive the stock price changes. Two datasets, the COVID data and the market data, are used. 

The report is decent and professional in the overall structure. The topics are interesting and worthy. I like the way the team solved the missing value problem by smoothing, which is suitable for the scenario. The team introduces the manipulation and exploration of the dataset detailedly, but still misses some info such as the observation window of the dataset, the features and the data size. The lack of introduction also puzzled a bit about model training logic. Lastly, the format of the report is so compact that it’s pretty hard to read. 

In short, the report looks comprehensive but many details are under concern. So before deploying the model, I'd like more pellucid explanation about the problems from the team.
